index.js

```diff
@@ -5,5 +5,8 @@ module.exports = {
   PaymentMedium: require('./src/payment-medium'),
   Quote: require('./src/quote'),
   Trade: require('./src/trade'),
-  Helpers: require('./src/helpers')
+  Helpers: require('./src/helpers'),
+  mocks: {
```

Comment:
Do mocks need to be exposed here? Would rather keep test code separate from src.